src/citrine/informatics/analysis_configuration.py

```diff
@@ -6,7 +6,30 @@
 
 
 class CrossValidationAnalysisConfiguration(Serializable['CrossValidationAnalysisConfiguration']):
-    """Configuration settings for running CV analysis in a Performance Workflow."""
+    """Configuration settings for running cross-validation in a performance workflow.
+
+    Parameters
+    ----------
+    name : str
+        Name of the analysis configuration
+    description: str
+        Description of the analysis configuration
+    n_folds: int
+        Number of k-folds
```

Comment:
Shouldn't this just be "Number of folds?" Also, although "k-folds" is common in literature, it might be confusing since it's not defined in this doc.